sqlstore/src/main/resources/db/migration/V0010__sample_hierarchy.sql

```diff
@@ -334,3 +334,85 @@ CREATE TABLE `SampleTissue` (
 
 ALTER TABLE Sample ADD COLUMN `sampleTissueId` BIGINT (20) DEFAULT NULL after sampleAnalyteId;
 ALTER TABLE Sample ADD FOREIGN KEY (sampleTissueId) REFERENCES SampleTissue (sampleTissueId);
+
+CREATE TABLE `SequencingParameters` (
+  `parameterId` bigint(20) PRIMARY KEY AUTO_INCREMENT,
+  `name` text NOT NULL,
+  `platformId` bigint(20) NOT NULL,
+  `createdBy` bigint(20) NOT NULL,
+  `creationDate` datetime NOT NULL,
+  `updatedBy` bigint(20) NOT NULL,
+  `lastUpdated` datetime NOT NULL,
+  CONSTRAINT `sequencingParameters_createUser_fkey` FOREIGN KEY (`createdBy`) REFERENCES `User` (`userId`),
+  CONSTRAINT `sequencingParameters_updateUser_fkey` FOREIGN KEY (`updatedBy`) REFERENCES `User` (`userId`)
+  -- Because Platform is MyISAM, we can't: CONSTRAINT `parameter_platformId_fkey` FOREIGN KEY (`platformId`) REFERENCES `Platform` (`platformId`)
+) ENGINE=InnoDB CHARSET=utf8;
+
+INSERT INTO `SequencingParameters` (`platformId`, `name`, `createdBy`, `updatedBy`, `creationDate`, `lastUpdated`)
+VALUES
+	(1,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(2,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(3,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(4,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(5,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(6,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(7,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(8,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(9,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(10,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(11,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(12,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(13,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(14,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(15,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v3 1×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v3 1×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v3 2×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v3 2×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v4 1×136', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v4 2×126', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'Rapid Run 1×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
```

Comment:
Any way to retrieve individual parameters? This is fine for human consumption, but for Pinery, and any other possible future MISO functions that consume this data, unkeyed parameter values concatenated in one field are not very useful